### PR TITLE
Update and rename README to README.md

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,28 +3,29 @@
 TO COMPILE PROJECTIONS:
```
Comment (matthiasdiener) [inline README.md]: Would suggest changing the case in this heading to title case or so. Also, please change to markdown headings, i.e. `# Compiling Projections`.

File: README.md [hunk 2]
```diff
@@ -3,28 +3,29 @@
...
+      
+      `projections.jar`       : archive of all the java and image files
 
 
 TO RUN PROJECTIONS:
 ------------------
 
-1.) From any location, type
-	> PROJECTIONS_LOCATION/bin/projections
+1.) From any location, type `> PROJECTIONS_LOCATION/bin/projections`
```
Comment (matthiasdiener) [inline README.md]: Please change `>` to `$`. I think three is no need for PROJECTIONS_LOCATION, could be just `$ ./bin/projections`

File: README.md [hunk 3]
```diff
@@ -3,28 +3,29 @@
...
 
-where PROJECTIONS_LOCATION is the path to the main projections
+where `PROJECTIONS_LOCATION` is the path to the main projections
 directory.
 
 HOW TO USE PROJECTIONS:
```
Comment (matthiasdiener) [inline README.md]: Maybe change '--enable-tracing' to `--enable-tracing` etc. below

File: README.md (hunk 2 above)
Comment (matthiasdiener) [reply README.md]: (in that case "From any location" should also be removed I guess.

Comment (matthiasdiener) [reply README.md]: No need for "1.)" as well I think

Comment (rbuch) [reply README.md]: Agreed, all done.

File: README.md [hunk 4]
```diff
@@ -1,44 +1,42 @@
 [![Build Status](https://travis-ci.org/UIUC-PPL/projections.svg?branch=master)](https://travis-ci.org/UIUC-PPL/projections)
 
-TO COMPILE PROJECTIONS:
+To Compile Projections:
```
Comment (matthiasdiener) [inline README.md]: Can you still change the heading from underline to `#` on the same line, ie. `# To Compile Projections`?

Comment (rbuch) [reply README.md]: Ah, yeah, good point, I'll do that.

**Pull request merged**